test/TestProjects/OperationGroupMappings/CodeModel.yaml

```diff
@@ -0,0 +1,948 @@
+!<!CodeModel> 
```

Comment:
for other examples this is inside the generated folder, any reason its here for this one?